src/dodal/devices/focusing_mirror.py

```diff
@@ -12,7 +19,7 @@
 DEFAULT_SETTLE_TIME_S = 60
 
 
-class MirrorStripe(Enum):
+class MirrorStripe(str, Enum):
```

Comment:
I feel conflicted about this. It's just far too pythony for me. But each to their own...

Reply:
It's an `ophyd_async` requirement that if you want to specify a signal as an `Enum` type it also has to inherit from `str`.